test/index.js

```diff
@@ -1269,8 +1416,11 @@ const HasManyDissociator = require('../src/services/has-many-dissociator');
               user: 'id,firstName,lastName,username,password,createdAt,updatedAt,resetPasswordToken',
             },
             page: { number: '2', size: '50' },
-            filterType: 'and',
-            filter: { username: '*hello*' },
+            filters: JSON.stringify({
+              field: 'username',
+              operator: 'contains',
+              value: 'hello',
+            }),
```

Comment:
Shouldn't we test with more complex conditions (at least several conditions with combined and/or)?